usr/share/rear/rescue/GNU/Linux/310_network_devices.sh

```diff
@@ -403,11 +408,19 @@ function ipaddr_setup () {
     if [ -n "$ipaddrs" ] ; then
         # If some IP is found for the network interface, then use them
         for ipaddr in $ipaddrs ; do
+            if grep -qw "$(echo "$ipaddr" | awk -F'/' '{ print $1 }')" <<< "${IGNORED_IP_ADDRESSES[@]}"; then
```

Comment:
I slightly prefer "${VAR[*]}" over "${VAR[@]}" in here strings to emphasize that we are interested in a single string, not multiple strings. In here strings it does not matter (I am not even sure why), but it makes the intent clearer.